telegram/_message.py

```diff
@@ -1581,6 +1592,8 @@ async def reply_text(
         chat_id, effective_reply_parameters = await self._parse_quote_arguments(
             do_quote, quote, reply_to_message_id, reply_parameters
         )
+        message_thread_id = message_thread_id or self.message_thread_id
```

Comment:
this could also be done directly in line 1609, but it doesn't matter much to me

Reply:
will do